parsers/sql.c

```diff
@@ -1004,8 +1004,9 @@ static void parseRecord (tokenInfo *const token)
 	 */
 	if (!isType (token, TOKEN_OPEN_PAREN))
 		readToken (token);
+	if (!isType (token, TOKEN_OPEN_PAREN))
+		return;
```

Comment:
yes, but the first advances token.  it's "if I'm called on a non-paren, then I advance, but if next isn't a paren either I give up"